library/std/src/Std/Math.qs

```diff
@@ -250,79 +250,85 @@ function Min(values : Int[]) : Int {
 
 /// # Summary
 /// Returns the angle whose cosine is the specified number.
+/// Returned value is expressed in radians.
 function ArcCos(x : Double) : Double {
     body intrinsic;
 }
 
 /// # Summary
 /// Returns the angle whose sine is the specified number.
+/// Returned value is expressed in radians.
 function ArcSin(y : Double) : Double {
     body intrinsic;
 }
 
 /// # Summary
 /// Returns the angle whose tangent is the specified number.
+/// Returned value is expressed in radians.
 function ArcTan(d : Double) : Double {
     body intrinsic;
 }
 
 /// # Summary
 /// Returns the angle whose tangent is the quotient of two specified numbers.
+/// Returned value is expressed in radians.
 function ArcTan2(y : Double, x : Double) : Double {
     body intrinsic;
 }
 
 /// # Summary
 /// Returns the cosine of the specified angle.
+/// Angle is specified in radians.
 function Cos(theta : Double) : Double {
     body intrinsic;
 }
 
 /// # Summary
-/// Returns the hyperbolic cosine of the specified angle.
```

Comment:
Is "angle" inaccurate? I know this PR is about specifying that we use radians, but radians are an expression of angle, so I would guess that "angle" is still appropriate.

Reply:
Could be that I'm missing some context regarding the hyperbolic trig functions though. It's been a while since I've learned about those.